scopes/workspace/workspace/types.ts

```diff
@@ -23,6 +23,11 @@ export interface WorkspaceExtConfig {
    */
   defaultDirectory: string;
 
+  /**
+   * sets the location of the root components directory.
```

Comment:
Add a note if this should be a path relative to the workspace / node_modules or an absolute path.
Also if you should have it with os specific path separator or linux format always